srv/components/s3server/config.sls

```diff
@@ -32,7 +30,8 @@ Stage - Config S3Server:
   cmd.run:
     - name: __slot__:salt:setup_conf.conf_cmd('/opt/seagate/cortx/s3/conf/setup.yaml', 's3:config')
 
+{% if pillar["cluster"][grains["id"]]["is_primary"] %}
```

Comment:
How is this related to this review?
We are not supposed to give any special treatment to any component. Thus, S3 should check if it should run on  a node or not.
Remove this block from here.